src/Compilers/CSharp/Test/Symbol/Symbols/Source/RecordTests.cs

```diff
@@ -805,7 +804,7 @@ .maxstack  2
 ");
         }
 
-        [Fact]
+        [Fact(Skip = "record struct")]
```

Comment:
nit: link to issue would be better. Or add note to test plan to unskip all tests.